### ISSUE TITLE
_saveScrollPosition call unnecessary unless opened is true

### ISSUE THREAD

**Issue by MKuenzi** [PolymerElements/iron-dropdown#112]
### Description

From what I can tell, I believe the call to _saveScrollPosition should only be called in _openedChanged if this.opened is actually true.  I have changed this locally and have not seen any issues, even if the iron-dropdown is within a scrolling container.

This would be a nice optimization because I am seeing this method cause a reflow, which may be completely unnecessary?
### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [X] IE 10

**Comment by web-padawan** [comment]
+1 to this issue. I see this in timeline:
```
Forced reflow is a likely performance bottleneck.
Self Time
27.50 ms
Total Time
27.50 ms
Elements Affected
1216
```

/cc @valdrinkoshi

**Comment by stat92** [comment]
Is anybody fixing this? _saveScrolPosition causing Recalculate Style even if the dropdown is closed. With fix #124 everything is OK, so why it has been closed?
